src/ngraph/op/max.cpp

```diff
@@ -82,6 +84,55 @@ shared_ptr<Node> op::v0::Max::get_default_value() const
     }
 }
 
+namespace
+{
+    template <element::Type_t ET>
+    bool evaluate(const HostTensorPtr& arg, const HostTensorPtr& out, const AxisSet& axes)
+    {
```

Comment:
Need to set the output shape/element type. See what `Add` does.

Reply:
Done